tools/gen-intrinsics-specs.py

```diff
@@ -800,9 +814,14 @@ def get_intrinsics_db(path):
     classification_map = get_classification_map(cli_args.classification)
     intrinsics_db = get_intrinsics_db(cli_args.intrinsic_defs)
     doc_template = read_template(cli_args.template)
-    intrinsic_table = process_db(intrinsics_db, classification_map)
-    rst_output = doc_template.format(intrinsic_table=intrinsic_table)
-    with (open(cli_args.outfile, 'w')) as f:
+    output_file = cli_args.outfile
+    file_format = cli_args.format
+    intrinsic_table = process_db(intrinsics_db, classification_map, file_format)
+    table_of_contents = ""
+    if cli_args.table_of_contents == "true":
+        table_of_contents = get_table_of_contents("")
```

Comment:
Please remove all the content related to the "Table of Contents" functionality, we will do it in a second step.